pyramid_zipkin/request_helper.py

```diff
@@ -119,13 +119,22 @@ def create_zipkin_attr(request):
     Attaches lazy attribute `zipkin_trace_id` with request which is then used
     throughout the tween.
 
+    consumes custom is_tracing function to determine if the request is traced
+    if one is set in the pyramid registry
+
     :param request: pyramid request object
     :rtype: :class:`pyramid_zipkin.request_helper.ZipkinAttrs`
     """
     request.set_property(get_trace_id, 'zipkin_trace_id', reify=True)
 
     trace_id = request.zipkin_trace_id
-    is_sampled = is_tracing(request)
+
+    settings = request.registry.settings
+    if 'zipkin.is_tracing' in settings:
+        is_sampled = settings['zipkin.is_tracing'](request)
+    else:
+        is_sampled = is_tracing(request)
```

Comment:
I'm not a huge fan of this way of overriding default behavior. I know it is done elsewhere in this project and I'm not helping since I don't know how it should look. But I feel like we need to figure out a better way to let users inject their own overrides. ¯\\\_(ツ)\_/¯